frontend/integration-tests/views/metalkube/dashboards.view.ts

```diff
@@ -0,0 +1,101 @@
+import { browser, element, by , $$ } from 'protractor';
+import { waitForNone } from '../../protractor.conf';
+
+export const untilNoLoadersPresent = waitForNone($$('.co-m-loader'));
+export const isLoaded = () => browser.wait(untilNoLoadersPresent).then(() => browser.sleep(2000));
+
+// Usage example:
+// await dashboardView.inventoryItemLabel(dashboardView.INVENTORY_NODES).getText()
+export function inventoryItemLabel(rowNumber) {
+  return element(by.xpath(`(//div[@class="kubevirt-inventory__row-title"])[${rowNumber + 1}]`));
+}
+
+// Usage example:
+// await dashboardView.inventoryUpCounter(dashboardView.INVENTORY_NODES).getText()
+export function inventoryUpCounter(rowNumber) {
+  // Explaining the xpath:
+  // There is an array of 6 kubevirt-inventory__row-status items (one for each line in the inventory).
+  // Then we look for a kubevirt-inventory__row-status-item which has a kubevirt-inventory__row-status-item-icon--ok
+  // under it, and go __back up__ to the parent (with "..") and find the span under it. This gives us
+  // the counter *next to* the kubevirt-inventory__row-status-item-icon--ok.
+  return element(by.xpath(`(//div[@class="kubevirt-inventory__row-status"])[${rowNumber + 1}]` +
+    '/div[contains(@class, "kubevirt-inventory__row-status-item")]' +
+    '/span[contains(@class, "kubevirt-inventory__row-status-item-icon--ok")]' +
+    '/../span[contains(@class, "kubevirt-inventory__row-status-item-text")]'));
+}
+
+// Usage example:
+// await dashboardView.inventoryDownCounter(dashboardView.INVENTORY_NODES).getText()
+export function inventoryDownCounter(rowNumber) {
+  // Similar xpath explanation to the one in inventoryUpCounter(), but now we're finding the counter
+  // next to the kubevirt-inventory__row-status-item-icon--error.
+  return element(by.xpath(`(//div[@class="kubevirt-inventory__row-status"])[${rowNumber + 1}]` +
+    '/div[contains(@class, "kubevirt-inventory__row-status-item")]' +
+    '/span[contains(@class, "kubevirt-inventory__row-status-item-icon--error")]' +
+    '/../span[contains(@class, "kubevirt-inventory__row-status-item-text")]'));
+}
+
+export const INVENTORY_NODES = 0;
```

Comment:
I think we should find the row in question instead. The order can change depending on things installed (e.g. kubevirt)